build.gradle

```diff
@@ -6,130 +6,130 @@ def VS_2015_LIB_DIR = ""
 
 allprojects{
   if(_MSC_VER == 1900){
-		VS_2015_INCLUDE_DIR = "C:/Program Files (x86)/Windows Kits/10/Include/10.0.10586.0/ucrt"
-		VS_2015_LIB_DIR = "C:/Program Files (x86)/Windows Kits/10/Lib/10.0.10586.0/ucrt"
-		}
-	apply plugin: 'cpp'
-	model{
-	  toolChains{
-			if(OperatingSystem.current().isWindows()){
-				visualCpp(VisualCpp){
-					if(_MSC_VER == 1900){
-						installDir "C:/Program Files (x86)/Microsoft Visual Studio 14.0"
-						}
-					else{
-						installDir "C:/Program Files (x86)/Microsoft Visual Studio 12.0"
-						}
-					}
-				}
-			else{
-				gcc(Gcc)
-				}
-			}
-			
-		if(OperatingSystem.current().isWindows()){
-			apply plugin: 'visual-studio'
-			if(_MSC_VER == 1900){
-				visualStudio{
-					solutions.all{
-						solutionFile.withContent{ TextProvider content ->
-						  content.asBuilder().insert(0, "# GENERATED FILE: DO NOT EDIT\n")
-						  content.text = content.text.replaceFirst("Format Version [1][1][.][0][0]", "Format Version 12.00")
-						  content.text = content.text.replaceFirst("[# ] Visual C[+][+] Express 2010", "# Visual Studio 14\nVisualStudioVersion = 14.0.25123.0\nMinimumVisualStudioVersion = 10.0.40219.1")
-						  }
-						}
-					projects.all{
-					  projectFile.withXml{
-						  asNode().@ToolsVersion = "14.0"
-							asNode().PropertyGroup.findAll({ it.@Label == 'Configuration'}).each{ configNode ->
-							  configNode.appendNode("PlatformToolset","v140")
-							  }
-						  }
-						}
-					}
-				}
-			else{
-				visualStudio{
-					solutionFile.withContent{ TextProvider content ->
-					  solutions.all
-						{
-						content.asBuilder().insert(0, "# GENERATED FILE: DO NOT EDIT\n")
-						content.text = content.text.replaceFirst("Format Version [1][1][.][0][0]", "Format Version 12.00")
-						content.text = content.text.replaceFirst("[# ] Visual C[+][+] Express 2010", "# Visual Studio 2013\nVisualStudioVersion = 12.0.40629.0\nMinimumVisualStudioVersion = 10.0.40219.1")
-						}
-					}
-					projects.all{
-						projectFile.withXml
-							{
-							asNode().@ToolsVersion = "12.0"
-							asNode().PropertyGroup.findAll({it.@Label == 'Configuration'}).each{ configNode ->
-							  configNode.appendNode("PlatformToolset","v120")
-							  }
-						  }
+    VS_2015_INCLUDE_DIR = "C:/Program Files (x86)/Windows Kits/10/Include/10.0.10586.0/ucrt"
+    VS_2015_LIB_DIR = "C:/Program Files (x86)/Windows Kits/10/Lib/10.0.10586.0/ucrt"
+    }
+  apply plugin: 'cpp'
+  model{
+    toolChains{
+      if(OperatingSystem.current().isWindows()){
+        visualCpp(VisualCpp){
+          if(_MSC_VER == 1900){
+            installDir "C:/Program Files (x86)/Microsoft Visual Studio 14.0"
             }
-				  }
-				}
-			}
-			
-		buildTypes{
-			Debug
-			Release
-			}
+          else{
+            installDir "C:/Program Files (x86)/Microsoft Visual Studio 12.0"
+            }
+          }
+        }
+      else{
```

Comment:
Indentation here is really off. Also, `else` should be on the same line as the closing brace.